packages/eslint-config-godaddy-flow/package-lock.json

```diff
@@ -494,16 +494,6 @@
         }
       }
     },
-    "eslint-config-godaddy": {
```

Comment:
Should we also kill the flow package? Are folks still using it?

Reply:
Sadly yes, in my org :(